kopf/clients/classes.py

```diff
@@ -0,0 +1,48 @@
+import json
+from typing import Type
+
+import pykube
+
+from kopf.clients import auth
+
+
+# TODO: this mixin has to be migrated into pykube-ng itself.
+class APIObject(pykube.objects.APIObject):
```

Comment:
I created https://github.com/hjacobs/pykube/pull/29 to have this upstream

Reply:
@hjacobs I've switched to `pykube-ng>=0.25`. So, the patching & CRD hacks are gone (including these APIObject/NamespacesAPIObject classes). See f105def0738a0e7dc43804ea7e4cf9cc9fff5a60